docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -364,40 +444,170 @@ Dagster Cloud currently includes support for four levels of role-based access co
+
+<Image
+alt="Override for a code location not allowed"
+src="/images/dagster-cloud/user-token-management/not-allowed-code-location-override.png"
+width={1035}
+height={466}
+/>
+
+To remove overrides:
+
+- **For all code locations in a deployment**, change the user's role at the deployment level. When saved, the new user role will be applied to all code locations in the deployment.
+- **For a single code location**, click **Edit user role** next to the code location to open the user role selection dialog. Then, click the **Remove override** button:
+
+  <Image
+  alt="Removing the override for a code location"
+  src="/images/dagster-cloud/user-token-management/remove-code-location-override.png"
+  width={536}
+  height={383}
+  />
+
+#### Branch deployment permissions
+
+<TODO></TODO>
+
+---
+
+## Adding users
+
+<Note>
+  <strong>Organization Admin</strong> or <strong>Admin</strong> permissions are
+  required to add users in Dagster Cloud.
+</Note>
+
+Before you start, note that:
+
+- **If using Google for SSO**, users must be added in Dagster Cloud before they can log in.
+
+- **If using a SAML-based solution like Okta**, users must be assigned to the Dagster app in the SSO portal to log in. By default, users will be granted Viewer permissions on each deployment. The default role can be adjusted by modifying the [`sso_default_role` deployment setting](/dagster-cloud/developing-testing/deployment-settings-reference#sso-default-role).
+
+1. Sign in to your Dagster Cloud account.
+2. Click the **user menu (your icon) > Cloud Settings**.
+3. Click the **Users** tab.
+4. In the **Enter an email...** field, enter the user's email address.
+5. Click **Add user**. The user will be added to the list of users.
+6. In the window that displays, select the appropriate [user role](#understanding-user-roles) for each deployment:
+   1. Next to a deployment, click **Edit user role**.
+   2. Select the user role for the deployment. This [user role](#understanding-user-roles) will be used as the default for all code locations in the deployment.
+   3. Click **Save**.
+   4. **Enterprise only**: To set permissions for individual code locations in a deployment:
+      1. Click the toggle next to the deployment.
+      2. Click **Edit user role**.
+      3. Select the user role for the code location.
+      4. Click **Save**.
+7. Repeat step 6 for each deployment.
+8. **Optional**: To change the user's permissions for branch deployments:
+   1. Next to **All branch deployments**, click **Edit user role**.
+   2. Select the user role to use for all branch deployments.
+   3. Click **Save**.
+9. Click **Done**.
+
+---
+
+## Removing users
+
+<Note>
+  <strong>Organization Admin</strong> or <strong>Admin</strong> permissions are
+  required to remove users in Dagster Cloud.
```

Comment:
Only Org Admins can remove (Admins can remove permissions for the things they have admin power over, but cannot remove the user entirely from the org)